content/200-orm/300-prisma-migrate/300-workflows/10-seeding.mdx

```diff
@@ -57,17 +57,18 @@ This can be useful to reduce memory usage (RAM) and increase execution speed of
 
 ## Integrated seeding with Prisma Migrate
 
-Database seeding happens in two ways with Prisma ORM: manually with `prisma db seed` and automatically in `prisma migrate dev` and `prisma migrate reset`.
+Database seeding happens in two ways with Prisma ORM: manually with `prisma db seed` and automatically in `prisma migrate reset` and (in some scenarios) `prisma migrate dev`.
 
 With `prisma db seed`, _you_ decide when to invoke the seed command. It can be useful for a test setup or to prepare a new development environment, for example.
 
-Prisma Migrate also integrates seamlessly with your seeds, assuming you follow the steps in the section below. When Prisma Migrate resets the development database, seeding is triggered automatically if you have a "seed" property in the "prisma" section in your package.json.
```

Comment:
The info about `package.json` was duplicated from before, and applies to all of seeding.